### PR TITLE
WR-364: Theme Icon Cards

### PR DESCRIPTION
**Pull request by zetagraph**
Tickets: 

https://hive.aved.gov.bc.ca/jira/browse/WR-364

https://hive.aved.gov.bc.ca/jira/browse/WR-167

### CODE REVIEW

File: src/web/themes/custom/workbc/templates/paragraph/paragraph--action-card-icon.html.twig [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ *
+ * @ingroup themeable
+ */
+#}
+
+{# Variables #}
+
+{% set card_w_icon_icon = content.field_icon[0] %}
+{% set card_w_icon_title = content.field_title[0] %}
+{% set card_w_icon_desc = content.	field_description[0] %}
```
Comment (infojunkie) [inline src/web/themes/custom/workbc/templates/paragraph/paragraph--action-card-icon.html.twig:46]: looks like a typo here

File: src/config/sync/core.entity_view_display.paragraph.action_card_icon.default.yml [hunk 2]
```diff
@@ -1,40 +1,36 @@
-uuid: a9b4fda1-ec26-4c5e-a1e4-978cab37ad92
+uuid: 9a5eee15-e10a-4b5b-9996-5ad7d4c87950
 langcode: en
 status: true
 dependencies:
   config:
-    - core.entity_view_mode.paragraph.action_card_icon
```
Comment (infojunkie) [inline src/config/sync/core.entity_view_display.paragraph.action_card_icon.default.yml:6]: do we need to delete existing content to sync this config change?

File: src/web/themes/custom/workbc/templates/paragraph/paragraph--action-card-icon.html.twig (hunk 1 above)
Comment (zetagraph) [reply src/web/themes/custom/workbc/templates/paragraph/paragraph--action-card-icon.html.twig]: Updated. Thanks. I will be doing another pass on this, markup and theming anyway. This was just the base setup.

File: src/config/sync/core.entity_view_display.paragraph.action_card_icon.default.yml (hunk 2 above)
Comment (zetagraph) [reply src/config/sync/core.entity_view_display.paragraph.action_card_icon.default.yml:6]: Don't think any icon cards have been added anywhere.

**Review by infojunkie — APPROVED** [review]
Lgtm

**Pull request merged**